action-scripts/merge_pr_jsons.py

```diff
@@ -125,11 +138,28 @@ def main():
             result["transactions"] = []
             for file in fs:
                 result["transactions"] += file["transactions"]
+                #  Check for gauge adds and generate checkpoint list
+
+                for tx in file["transactions"]:
+                    if tx["contractMethod"]["name"] == "addGauge":
+                        try:
+                            gauge_chain = tx["contractInputsValues"]["gaugeType"]
+                            if gauge_chain != "Ethereum":
```

Comment:
What's the point of this check here?

Reply:
1: We don't want to add Ethereum gauges to the L2 checkpointer.
2: We don't want to act unless it is an addGauge
3: There are older versions and maybe newer versions of the gaugeAdder that don't have gaugeType, and we don't want to add those.

Reply:
Gotcha